extensions/prototypes/labgraph_monitor/README.md

```diff
@@ -21,28 +21,31 @@ yarn -v
 
 **Set up the application**
 
+(!) The following tutorial utilizes **yarn** (recommended) however **npm** can be utilized as well.
+
 1. Be sure that you are inside **extensions/prototypes/labgraph_monitor** directory
 
 ```
-cd extensions/prototypes/labgraph_monitor
+labgraph> cd extensions/prototypes/labgraph_monitor
+labgraph\extensions\prototypes\labgraph_monitor>
 ```
 
 2. Install dependencies by running **yarn** command
 
 ```
-yarn
+labgraph\extensions\prototypes\labgraph_monitor> yarn
 ```
 
 3. Test the application by running the following command
 
 ```
-yarn test --watchAll=false
+labgraph\extensions\prototypes\labgraph_monitor> yarn test --watchAll=false
```

Comment:
The following error appears when running this command: https://paste.ofcode.org/VFn3XgZGCsnF3hiTGKNXuZ

Reply:
I resolved it, thank you for pointing that out